### PR TITLE
Expose animation handling

### PR DESCRIPTION
**Pull request by yuryybk**
This PR resolves mapbox/mapbox-gl-native-android#467

### CODE REVIEW

File: MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java [hunk 1]
```diff
@@ -2116,6 +2116,17 @@ public LocationComponent getLocationComponent() {
...
+   * Tells the map rendering engine that the animation is currently performed by the
+   * user (e.g. with a `jumpTo()` calls series). It adjusts the engine for the animation use case.
+   * In particular, it brings more stability to symbol placement and rendering.
+   *
+   * @param inProgress Bool representing if user animation is in progress
+   */
+   public void setUserAnimationInProgress(boolean inProgress) {
+     nativeMapView.setUserAnimationInProgress(inProgress);
+   }
+
```
Comment (Chaoba) [inline MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java:2129]: Miss `isUserAnimationInProgress` here.

Comment (yuryybk) [reply MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java:2129]: yeah, I've exposed only `setUserAnimationInProgress` in the same way as in Carbon, https://github.com/mapbox/mapbox-maps-android/blob/master/sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

should we expose  `isUserAnimationInProgress` only here? or it is missed in Carbon too?

Comment (tobrun) [reply MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java:2129]: We need both getter as setter, need to double check carbon impl but that should have both afaik

Comment (yuryybk) [reply MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMap.java:2129]: exposed `isUserAnimationInProgress`

**Pull request merged**